scripts/process_data.py

```diff
@@ -801,6 +880,64 @@ def main(self) -> None:
         CONSOLE.rule()
 
 
+@dataclass
+class ProcessRecord3D:
+    """Process Record3D data into a nerfstudio dataset.
+
+    This script does the following:
+
+    1. Scales images to a specified size.
+    2. Converts Record3D poses into the nerfacto format.
+    """
+
+    data: Path
+    """Path the data, either a video file or a directory of images."""
+    output_dir: Path
+    """Path to the output directory."""
+    num_downscales: int = 3
+    """Number of times to downscale the images. Downscales by 2 each time. For example a value of 3
+        will downscale the images by 2x, 4x, and 8x."""
+    verbose: bool = False
```

Comment:
We should specify a target number of images. otherwise these datasets get quite large, right?

Reply:
I was thinking this would be better put in the nerfacto dataparser. Since then you could scale the number of training images you use without having to go back to the process step.

Reply:
Hmm, that is a good point. The one thing I'm worried about his how we set defaults. Because if a user explicitly processes one of other "image" or "videos" type dataset to have lots of images, it may be counterintuitive if not all of the images are used due to a default set in the dataparser.

Reply:
I could see either way-- Maybe we could just have a reasonably high default count in the nerfacto dataparser, like 1k or something.

Reply:
Does record3d perform well with 1k images?

Reply:
I can check-- I doubt it hurts performance but it might just take a long time for everything to load in and doesn't give much gains